app/templates/CommentList.rss.twig

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<rss version="2.0">
+    <channel>
+        <title>Spenden an Wikimedia</title>
+        <link>{$ rssLink |e $}</link>
```

Comment:
All variables are [auto-escaped for HTML output](http://twig.sensiolabs.org/doc/api.html#environment-options). Adding `|e` to all placeholders is unneccessarily explicit and verbose IMHO.